tests/importer/importer.cpp

```diff
@@ -76,10 +79,13 @@ TEST(Importer, warningCircularImportReferencesUnits)
 {
     const std::string errorMessage =
         "Cyclic dependencies were found when attempting to resolve units in the model 'circularImport1'. The dependency loop is:\n"
-        " - units 'i_am_cyclic' is imported from 'u2' in 'circularUnits_2.cellml';\n"
-        " - units 'u2' is imported from 'u3' in 'circularUnits_3.cellml';\n"
-        " - units 'u3' is imported from 'i_am_cyclic' in 'circularUnits_1.cellml'; and\n"
-        " - units 'i_am_cyclic' is imported from 'u2' in 'circularUnits_2.cellml'.";
+        " - units 'i_am_cyclic' specifies an import from ':this:' to '"
+        + resourcePath("importer/") + "circularUnits_2.cellml';\n"
+                                      " - units 'u2' specifies an import from '"
+        + resourcePath("importer/") + "circularUnits_2.cellml' to '" + resourcePath("importer/") + "circularUnits_3.cellml'; and\n"
+                                                                                                   " - units 'u3' specifies an import from '"
+        + resourcePath("importer/") + "circularUnits_3.cellml' to '" + resourcePath("importer/") + "circularUnits_1.cellml'.";
+
```

Comment:
Replace with:
```c++
std::string("Cyclic dependencies were found when attempting to resolve units in the model 'circularImport1'. The dependency loop is:\n")
+ " - units 'i_am_cyclic' specifies an import from ':this:' to '" + resourcePath("importer/") + "circularUnits_2.cellml';\n"
+ " - units 'u2' specifies an import from '" + resourcePath("importer/") + "circularUnits_2.cellml' to '" + resourcePath("importer/") + "circularUnits_3.cellml'; and\n"
+ " - units 'u3' specifies an import from '" + resourcePath("importer/") + "circularUnits_3.cellml' to '" + resourcePath("importer/") + "circularUnits_1.cellml'.";
```
to make it much easier (for me at least!) to read.

Reply:
see above.

Reply:
Likewise. :)